### ISSUE TITLE
Automap is a terrible idea

### ISSUE THREAD

**Issue by ghost** [AryanWolf3D/Wolf4SDL#28]
So I was adding KyleRTCW's spotsaw stuff here:
http://dhw.wolfenstein3d.com/viewtopic.php?p=77859#77859

And realized: "man, this isn't wolf3d anymore... this is some lameass jpb-tier cheating bullshit!". Also, if the guards you can't see are still shown, it's another form of cheating by ruining the surprise. This automap feature feels like opening a can of worms into hell. I think a better idea is to just keep it a Tab-O debug cheat that can see everything (all enemies and sprites) etc. and fuck automap, fuck ecwolf, fuck Blzut3, and fuck Zombie. I miss Zombie being healthy.

TL;DR Wolf3d is better as memory game or surprise game so fuck automap. At the very least I think if the "automap" is a non-cheating normal feature in Wolf4SDL rather than a debug cheat then the #define VIEWMAP in version.h should be off/commented out by default. It's REALLY TERRIBLE to have this as a normal part of wolf3d! Imagine people actually using it when playing the game normally. Having those NO NO NO NO! Tile16M images being shown for the entire 64x64 grid would be better or a message saying "you are a lamer". 🥂

**Comment by KS-Presto** [comment]
Righto, I guess Blake Stone, Rise of the Triad, and Doom are terrible games too then.

⫷ °⧭° ⫸
